cognee/tests/integration/run_toy_tasks/conftest.py

```diff
@@ -0,0 +1,11 @@
+import os
+
+import pytest
+
+
+@pytest.fixture(autouse=True, scope="session")
+def copy_cognee_db_to_target_location():
```

Comment:
Why do we copy the database? Cognee will create a new one in each run, we just need to set system path like we already do in end-to-end tests.

Reply:
This turned out to be necessary for the integration tests to run, I don't know the library well enough to know why exactly that is the case. But it looks for this file at this location when querying for a user, and errors out if it doesn't find it. 

I tried monkeypatching the database but that didn't work unfortunately.